diff --git a/lib/esm/parse.js b/lib/esm/parse.js
index 781d6a47224bf5c848198372b44571cb6545283f..4cb0d70db5382fa3b7919356b5f62cf15d83f280 100644
--- a/lib/esm/parse.js
+++ b/lib/esm/parse.js
@@ -17,9 +17,9 @@ export function getParse(parser) {
      * @returns The parsed document node.
      */
     return function parse(content, options, isDocument, context) {
-        if (typeof Buffer !== 'undefined' && Buffer.isBuffer(content)) {
-            content = content.toString();
-        }
+        // if (typeof Buffer !== 'undefined' && Buffer.isBuffer(content)) {
+        //     content = content.toString();
+        // }
         if (typeof content === 'string') {
             return parser(content, options, isDocument, context);
         }
